src/Core/Framework/Plugin/PluginManagementService.php

```diff
@@ -113,18 +107,11 @@ public function downloadStorePlugin(PluginDownloadDataStruct $location, Context
         }
 
         $this->extractPluginZip($tempFileName, true, $location->getType());
-
-        if ($location->getType() === self::PLUGIN) {
-            $this->pluginService->refreshPlugins($context, new NullIO());
```

Comment:
`tests/e2e/cypress/e2e/installer/install_nl.cy.js` failes while doing the onboarding. Because the plugins aren't refreshed, the installation of the PayPal plugin failes with `"Plugin by name "SwagPayPal" not found."`.
Please take a look and fix this.

Reply:
Where did you see the execution of this cypress test? Did it run in the CI? If so, i cant find it, as the cypress CI in this PR fails with another, unrelated error...

Reply:
Sadly the Github pipeline does not run all cypress tests, you have to run them in your test environment.
But you can reproduce this manually by resetting your environment: Drop your database, clear the cache (`var/cache`) and delete the `install.lock` (if present). Then go trough the onboarding and run the "First Run Wizard" - you can find the wizard in the Settings > System module, if not presented automatically. While skipping trough the steps, take a look at the network traffic and wait for the failing request on the PayPal configuration step.

Reply:
Is there a reason you removed all the `refreshPlugins()` calls? we just found out that this calls are now missing for plugin installation through the extension store as well and failing with the same error like the test failure here, so we will bring back the the `refreshPlugin` calls here for now

any points @maximilianruesch do you see anything why we shouldn't do the refresh call here?

cc @fschmtt